lets-talk.fifthtry.site/js/index.mjs

```diff
@@ -52,13 +52,99 @@ function UTCDateStringToFormattedString(dateString) {
     return formatted;
 }
 
-function convertToString(datetime) {
-    const date = datetime.get().toObject().dt;
-    return date.toString();
+function to_timestamp_millis(date, time) {
+    const dateStr = date.toString();
+        const year = parseInt(dateStr.slice(0, 4));
+        const month = parseInt(dateStr.slice(4, 6)) - 1; // JS months are 0-based
+        const day = parseInt(dateStr.slice(6, 8));
+
+        // Parse time: nanoseconds since midnight
+        const nanosSinceMidnight = BigInt(time);
+
+        // Calculate hours, minutes, seconds and milliseconds
+        const nanosPerSecond = 1_000_000_000n;
+
+        const totalSeconds = Number(nanosSinceMidnight / nanosPerSecond);
+        const hour = Math.floor(totalSeconds / 3600);
+        const minute = Math.floor((totalSeconds % 3600) / 60);
+        const second = totalSeconds % 60;
+        const millisecond = Math.floor(
+            Number(nanosSinceMidnight % nanosPerSecond) / 1_000_000
+        );
+
+        // Construct JS Date (ms precision)
+        const dateObj = Date.UTC(
+            year,
+            month,
+            day,
+            hour,
+            minute,
+            second,
+            millisecond
+        );
+
+        return dateObj;
 }
 
+function schedule_meeting(title, start_datetime, end_datetime, meeting_attendees) {
```

Comment:
Ideally we should make backend request using ftd.http etc, not via fetch() call.